docs/sources/alerting/configure-notifications/template-notifications/reference.md

```diff
@@ -96,25 +96,25 @@ You can execute this template by passing the dot (`.`):
 
 `Alert` contains data for an individual alert:
 
-| Name           | Type          | Description                                                                                                                                    |
-| -------------- | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
-| `Status`       | string        | Firing or resolved.                                                                                                                            |
-| `Labels`       | [KV](#kv)     | The labels for this alert. It includes all [types of labels](ref:label-types).                                                                 |
-| `Annotations`  | [KV](#kv)     | The annotations for this alert.                                                                                                                |
-| `StartsAt`     | [Time](#time) | The time the alert fired                                                                                                                       |
-| `EndsAt`       | [Time](#time) | Only set if the end time of an alert is known. Otherwise set to a configurable timeout period from the time since the last alert was received. |
-| `GeneratorURL` | string        | A link to Grafana, or the source of the alert if using an external alert generator.                                                            |
-| `Fingerprint`  | string        | A unique string that identifies the alert.                                                                                                     |
+| Name           | Type          | Description                                                                                                                                                          |
+| -------------- | ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `Status`       | string        | Firing or resolved.                                                                                                                                                  |
+| `Labels`       | [KV](#kv)     | The labels associated with this alert. <br/> It includes all [types of labels](ref:label-types), but only those used in evaluating the alert condition are included. |
```

Comment:
That's correct but I can't think of the opposite example. What labels do you mean under labels that are not used in evaluating?

Reply:
Honestly, I could not replicate this https://community.grafana.com/t/grafana-11-3-how-to-make-more-labels-available-for-alert-notification/144468 but I thought it was related to this case

Reply:
Yea, there is something wrong with the customer's query. Server-side expressions do not drop\add any labels to the result returned by data source

Reply:
> The labels associated with this alert. It includes all [types of labels](https://grafana.com/docs/grafana/latest/alerting/fundamentals/alert-rules/annotation-label/#label-types), but only query labels used in the alert condition.

I changed the `Labels` description a bit. If mentioning the alert condition for labels is unnecessary or misleading, it's better to leave it as before:

> The labels associated with this alert. It includes all [types of labels](https://grafana.com/docs/grafana/latest/alerting/fundamentals/alert-rules/annotation-label/#label-types).

Reply:
@yuri-tceretian pls, help me on deciding this copy so we can move forward with the PR.  Thanks!